build/common.targets

```diff
@@ -15,6 +15,10 @@
     <DefineConstants>$(DefineConstants);NETCORE5_0</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == '$(NETFXTargetFramework)' OR '$(TargetFramework)' == 'netcoreapp5.0'">
```

Comment:
Good suggestion! Thanks! So I just list the TFMs not supporting signing(as it won't change, compared to the TFMs supporting signing), then define constant IS_SIGNING_SUPPORTED when the TFM is not in the list. I think it will be future-proof then :)